homeassistant/util/yaml.py

```diff
@@ -16,6 +16,7 @@
 _LOGGER = logging.getLogger(__name__)
 _SECRET_NAMESPACE = 'homeassistant'
 _SECRET_YAML = 'secrets.yaml'
+_SECRET_CACHE = {}  # type: Dict
```

Comment:
We originally added it to the loader so that secrets is not visible by all after initializing, but see now that there is a new loader for each file, so this is not working as expected.
Maybe call this `__SECRET_CACHE` and it should be more difficult to access from outside?